### PR TITLE
Garima changes

### PR DESCRIPTION
**Pull request by Garima912**
I fixed the default and new looks.
Also I made score values look bold.

### CODE REVIEW

File: src/main/java/Util.java [hunk 1]
```diff
@@ -10,12 +10,11 @@
     static int height = 600;
     static int spaceBtwRows = 15;
     static int sidePadding = 10;
-    static int drawingBoxLeftPadding = 410;
+    static int drawingBoxLeftPadding = 320;
```
Comment (appleEyeBall) [inline src/main/java/Util.java]: Why did you change these values?

**Review by appleEyeBall — CHANGES_REQUESTED** [review]
Nice so far. just click resolve conversation and i'll merge

File: src/main/java/Util.java (hunk 1 above)
Comment (Garima912) [reply src/main/java/Util.java]: Because after making the score value bold, "Number of spots" was not fitting in the screen. 
You can try using 410 , you'll understand better what I'm saying

**Review by appleEyeBall — APPROVED** [review]

**Comment by appleEyeBall** [comment]
- Garima: Looks and theme
- Tise: animation and play/pause button
